enterprise/frontend/src/embedding-sdk-bundle/lib/sdk-question/update-question.ts

```diff
@@ -96,7 +96,11 @@ export const updateQuestionSdk =
     }
 
     const metadata = getMetadata(getState());
-    nextQuestion = new Question(nextQuestion.card(), metadata);
+    nextQuestion = new Question(
+      nextQuestion.card(),
+      metadata,
+      nextQuestion._parameterValues,
```

Comment:
This line is important, it properly sets entered in UI params to the new question